test/integration/simple_example/controls/gcloud.rb

```diff
@@ -34,6 +34,7 @@
     its(:stderr) { should eq "" }
     its(:stdout) { should include "example-dicom-a" }
     its(:stdout) { should include "example-dicom-b" }
+    its(:stdout) { should include "example-dicom-c" }
```

Comment:
can you also add a check for the streamConfigs being set as what we've done for FHIR at line 66?